packages/cln_plugin/lib/src/rpcmethods/init/init.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+
+class InitMethod extends AbstractRpcMethod {
+  InitMethod() : super(name: 'init', usage: '', description: '');
```

Comment:
Mh!

Now the question is how we can give meaning to this class? for sure, it needs a callback, but in this case, making a callback inside the constructor of the init method is painful.

I will think about this and post a solution for tomorrow :)